studio/components/layouts/ProjectLayout/LayoutHeader/FeedbackDropdown/FeedbackWidget.tsx

```diff
@@ -118,7 +118,7 @@ const FeedbackWidget = ({
           pathname: router.asPath,
         })
         setFeedback('')
-        ui.setNotification({ category: 'success', message: 'Feedback sent. Thank you!' })
+        ui.setNotification({ category: 'success', message: 'Feedback sent. Thank you!\n\nKindly be aware that we do not provide responses to feedback. If you require assistance or a reply, kindly consider submitting a support ticket.' })
```

Comment:
Nit: You used "Kindly" twice in a row

```suggestion
        ui.setNotification({ category: 'success', message: 'Feedback sent. Thank you!\n\nKindly be aware that we do not provide responses to feedback. If you require assistance or a reply, consider submitting a support ticket.' })
```